### PR TITLE
fix url in quickstart

### PR DESCRIPTION
**Pull request by wuxiangzhou2010**

### CODE REVIEW

File: content/ja/docs/quickstart/_index.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ $ govendor fetch github.com/gin-gonic/gin@v1.3
 4. 開始用テンプレートをプロジェクトディレクトリにコピーする
 
 ```sh
-$ curl https://raw.githubusercontent.com/gin-gonic/gin/master/examples/basic/main.go > main.go
+$ curl curl https://raw.githubusercontent.com/gin-gonic/examples/master/basic/main.go > main.go > main.go
```
Comment (thinkerou) [inline content/ja/docs/quickstart/_index.md]: two curl.

File: content/zh-cn/docs/quickstart/_index.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ $ govendor fetch github.com/gin-gonic/gin@v1.3
 4. Copy a starting template inside your project
 
 ```sh
-$ curl https://raw.githubusercontent.com/gin-gonic/gin/master/examples/basic/main.go > main.go
+$ curl curl https://raw.githubusercontent.com/gin-gonic/examples/master/basic/main.go > main.go > main.go
```
Comment (thinkerou) [inline content/zh-cn/docs/quickstart/_index.md]: two curl.

File: content/zh-tw/docs/_index.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ $ govendor fetch github.com/gin-gonic/gin@v1.3
 4. 复制启动文件模板到项目目录中
 
 ```sh
-$ curl https://raw.githubusercontent.com/gin-gonic/gin/master/examples/basic/main.go > main.go
+$ curl curl https://raw.githubusercontent.com/gin-gonic/examples/master/basic/main.go > main.go > main.go
```
Comment (thinkerou) [inline content/zh-tw/docs/_index.md]: two curl.

File: content/zh-tw/docs/quickstart/_index.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ $ govendor fetch github.com/gin-gonic/gin@v1.3
 4. Copy a starting template inside your project
 
 ```sh
-$ curl https://raw.githubusercontent.com/gin-gonic/gin/master/examples/basic/main.go > main.go
+$ curl curl https://raw.githubusercontent.com/gin-gonic/examples/master/basic/main.go > main.go > main.go
```
Comment (thinkerou) [inline content/zh-tw/docs/quickstart/_index.md]: two curl.

File: content/zh-cn/docs/quickstart/_index.md (hunk 2 above)
Comment (wuxiangzhou2010) [reply content/zh-cn/docs/quickstart/_index.md]: fixed with 9f0edca , please review

**Pull request merged**